Code/Engine/InputManager.cpp

```diff
@@ -113,62 +140,4 @@ float sad::InputManager::GetAxis(SDL_GameControllerAxis axis)
     roundedAxis = std::ceil(roundedAxis * 10.0) / 10.0;
```

Comment:
Probably want to avoid the implicit cast from `double` to `float` here to avoid warnings.

```
roundedAxis = static_cast<float>(std::ceil(roundedAxis * 10.0) / 10.0);
```